Patch/code review (#1)
keptn-sandbox/ansibletower-service: README.md

    keptn send event new-artifact --project=sockshop --service=carts --image=docker.io/keptnexamples/carts --tag=0.11.1
    ```

1. Next we are going to write a simple remediation file that we are adding to Keptn. Please copy the following contents into a file called `remediation-tower.yaml`. Please refer to https://keptn.sh/docs/concepts/automated_operations for more information.

    Please note to change the value of the `JobTemplate` to trigger to the value of your job template in Ansible Tower.

    ```
    apiVersion: spec.keptn.sh/0.1.4
    kind: Remediation
    metadata:
    name: remediation-service
    spec:
    remediations:
        - problemType: ansibletower service test event
        actionsOnOpen:
            - name: Test event
            description: Testing the Ansible Tower integration
            action: job_template_launch
            value:
                JobTemplate: "9"
    ```

1. Add remediation file to keptn.

    ```
    keptn add-resource --project=sockshop --stage=production --service=carts --resource=remediation-tower.yaml --resourceUri=remediation.yaml
    ```

>     Note: the name of the remediation file must be `remediation.yaml`.

Reviewer: @jetzlstorfer the way you rewrote using `remediation-tower.yaml` as file name was much better but now the Note here is confusing. How about we change it to: 
```suggestion
    Note: it is mandatory that the `resourceUri` parameter be exactly `remediation.yaml`
```

1.  Dummy test event

    Save the contents to a file, it doesn't matter which name you use.
    ```
    {
        "type": "sh.keptn.event.problem.open",
        "specversion": "0.2",
        "source": "https://github.com/keptn/keptn/dynatrace-service",
        "id": "f2b878d3-03c0-4e8f-bc3f-454bc1b3d79d",
        "time": "2020-06-07T07:02:15.64489Z",
        "contenttype": "application/json",
        "data": {
            "State": "OPEN",
            "ProblemID": "ab81-941c-1245",
            "ProblemTitle": "ansibletower service test event",
            "project": "sockshop",
            "stage": "production",
            "service": "carts"
        }
    }
    ```

    Note that the `ProblemTitle` value here must exactly match the `problemType` value in the remediation file.

1. Send event

    Send the event to trigger the remediation action.

    ```
    keptn send event --file=./(your test file).json
    ```

    Note: change the value between parentesis to match the file name you used for the dummy event.

> 1.  Optionally you can validate the the logs of the `ansibletower-service` pod.

Reviewer: ```suggestion
1.  Optionally you can validate the logs of the `ansibletower-service` pod.
```
